### ISSUE TITLE
Testing and unavailable dates

### ISSUE THREAD

**Issue by charliesmitty** [greatfire/applecensorship#6]
This app has been tested since February 2022 but is labelled as being unavailable for the first time in 2019. How is that possible?

![tested since](https://user-images.githubusercontent.com/7923756/156160838-fcbbeac5-e0f2-4207-816f-1cf24474d94d.png)

**Comment by charliesmitty** [comment]
Referring to the iHeart Radio app

**Comment by greatfire-martin** [comment]
Yeah that's a bug. Fixed now

**Comment by charliesmitty** [comment]
Is that fixed by removing the "Tested since" column?

**Comment by greatfire-martin** [comment]
No, I still see the "Tested since" column, and now it should be displaying the correct value
